application/src/main/java/be/jpendel/application/PersonMapper.java

```diff
@@ -6,7 +6,9 @@
 import java.util.List;
 import java.util.stream.Collectors;
 
-public class PersonMapper {
+/* TODO : mark: PersonMapper en PersonDTO moeten volgens mij in de domain layer
+*/
+class PersonMapper {
```

Comment:
De PersonDTO is het application contract, en de application layer is de enige waar dat we weten wat zowel het intern domein de application api kent, dus volgens mij moeten we dat niet dieper schuiven.

U application layer in dit geval bekijken als de anti-corruption layer van het domain naar buiten toe helpt misschien.